packages/components-native/src/Typography/Typography.tsx

```diff
@@ -334,12 +335,14 @@ export type TextColor =
 export type TextVariation =
   | "success"
   | "interactive"
+  | "interactiveSubtle"
```

Comment:
Extending the semantic variations here is part of https://jobber.atlassian.net/browse/JOB-97200